src/aviary/utils.py

```diff
@@ -79,17 +80,35 @@ def is_coroutine_callable(obj) -> bool:
     return False
 
 
-async def eval_answer(
+async def eval_answer(  # noqa: PLR0911
     proposed: str,
     correct: str,
     question: str | None = None,
-    eval_mode: EvalAnswerMode = EvalAnswerMode.CONTAINS,
+    eval_mode: str
+    | EvalAnswerMode
+    | Sequence[str | EvalAnswerMode] = EvalAnswerMode.CONTAINS,
     llm_eval_config: dict | None = None,
 ) -> float:
     """Evaluate a proposed answer against a correct answer.
 
     Will return 0 or 1, except for llm-score which should be between 0 and 1
+
+    If a sequence of modes is provided, we will return the first non-zero result.
+    Can be used to try a fast (e.g. EM) mode and fall back to a slow (e.g. LLM) mode.
     """
+    if isinstance(eval_mode, str):
+        eval_mode = EvalAnswerMode(eval_mode)
+
+    if not isinstance(eval_mode, EvalAnswerMode):
+        # Note we cannot check isinstance(..., Sequence) since StrEnums are sequences.
+        for mode in eval_mode:
+            result = await eval_answer(
+                proposed, correct, question, mode, llm_eval_config
+            )
+            if result:
+                return result
+        return 0.0
```

Comment:
Can you be explicit in the truthiness here? Like is this `result is not None`?

Reply:
I'm worried this will bias things - where if exact match says incorrect, it will get another try to use LLM eval, which could bias as a weird kind of pass@k-eval-methods